include/librealsense2/hpp/rs_frame.hpp

```diff
@@ -90,6 +156,11 @@ namespace rs2
             error::handle(e);
             return res;
         }
+		/**
+		* Register the extrinsic to the environment base on another stream profile
```

Comment:
Need to be clarified. The function assign extrinsic transformation parameters to a specific profile (sensor).
The extrinsic information is generally available as part of the camera calibration, and librealsense is responsible to retrieve and assign these parameters where appropriate.
The specific function is intended for synthetic/mock-up (software) devices for which the parameters are produced and injected by the user.